src/terrain/generation/land_generator.cpp

```diff
@@ -128,7 +131,7 @@ JsonToTile::get_volume(
             y_max--;
     }
 
-    TerrainOffset z_max = rand() % (Dz + 1) + Sz - Dz / 2;
+    TerrainOffset z_max = rand() % (height_variance + 1) + height - height_variance / 2;
```

Comment:
>There are no guarantees as to the quality of the random sequence produced. In the past, some implementations of `rand()` have had serious shortcomings in the randomness, distribution and period of the sequence produced (in one well-known example, the low-order bit simply alternated between 1 and 0 between calls).
> `rand()` is not recommended for serious random-number generation needs. It is recommended to use C++11's [random number generation](https://en.cppreference.com/w/cpp/numeric/random) facilities to replace `rand()`. (since C++11)

https://en.cppreference.com/w/cpp/numeric/random

Reply:
Create an issue

Reply:
No you